cluster/hooks.go

```diff
@@ -443,6 +445,21 @@ func InstallKubernetesDashboardPostHook(input interface{}) error {
 			return err
 		}
 
+		// set admin rights (if needed)
+		if cluster.NeedAdminRights() {
+
+			userId, err := cluster.GetKubernetesUserID()
+			if err != nil {
+				return err
+			}
+
+			if err := setAdminRights(client, userId); err != nil {
```

Comment:
This is confusing as `cluster.GetKubernetesUserID()` returns a user id while `setAdminRights` expects `userName`.